public/components/trace_analytics/components/traces/traces_content.tsx

```diff
@@ -109,34 +118,60 @@ export function TracesContent(props: TracesProps) {
 
   return (
     <>
-      <SearchBar
-        query={query}
+      <EuiFlexGroup
+        gutterSize="s"
+        alignItems="center"
+        justifyContent="spaceBetween"
+        style={{ padding: '0 16px' }}
+      >
+        <EuiFlexItem grow={false}>
+          <DataSourcePicker
+            modes={props.modes}
+            selectedMode={props.mode}
+            setMode={props.setMode!}
+          />
+        </EuiFlexItem>
+        <EuiFlexItem grow={true}>
+          <SearchBar
+            query={query}
+            filters={filters}
+            appConfigs={appConfigs}
+            setFilters={setFilters}
+            setQuery={setQuery}
+            startTime={startTime}
+            setStartTime={setStartTime}
+            endTime={endTime}
+            setEndTime={setEndTime}
+            refresh={refresh}
+            page={page}
+            mode={mode}
+            attributesFilterFields={attributesFilterFields}
+          />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+      <Filters
+        page={page}
         filters={filters}
-        appConfigs={appConfigs}
         setFilters={setFilters}
-        setQuery={setQuery}
-        startTime={startTime}
-        setStartTime={setStartTime}
-        endTime={endTime}
-        setEndTime={setEndTime}
-        refresh={refresh}
-        page={page}
+        appConfigs={appConfigs}
         mode={mode}
         attributesFilterFields={attributesFilterFields}
       />
       <EuiSpacer size="m" />
-      <EuiPanel>
-        <EuiAccordion
-          id="accordion1"
-          buttonContent={mode === 'data_prepper' ? 'Trace Groups' : 'Service and Operations'}
-          forceState={trigger}
-          onToggle={onToggle}
-          data-test-subj="trace-groups-service-operation-accordian"
-        >
-          <EuiSpacer size="m" />
-          {trigger === 'open' && dashboardContent()}
-        </EuiAccordion>
-      </EuiPanel>
+      <div style={{ padding: '0 16px' }}>
```

Comment:
any reason why we have to use div instead of OUI component?

Reply:
Switched too `<EuiPage paddingSize='m'>`